src/dc_context.c

```diff
@@ -15,6 +15,35 @@
 #include "dc_apeerstate.h"
 
 
+static const char* config_keys[] = {
+	"addr",
+	"mail_server",
+	"mail_user",
+	"mail_pw",
+	"mail_port"
+	"send_server",
+	"send_user",
+	"send_pw",
+	"send_port"
+	"server_flags",
+	"displayname"
+	"selfstatus",
+	"selfavatar",
+	"e2ee_enabled",
+	"mdns_enabled",
+	"save_mime_headers",
+};
```

Comment:
as mentioned in the additional comment, maybe we should add the `configured*` keys here